### PR TITLE
migrate 4.14 gcp testconfigs to multi-arch file

### PR DESCRIPTION
**Pull request by lwan-wanglin**
refer [OCPQE-17594](https://issues.redhat.com/browse/OCPQE-17594)

migrate 4.14 gcp testconfigs to multi-arch file
remove uncleaned jobs for 4.15 gcp on amd64
add stable test for 4.14 amd/arm/mutl

### CODE REVIEW

**Comment by lwan-wanglin** [comment]
@jianlinliu @aleskandro PTAL! thanks!

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-stable.yaml [hunk 1]
```diff
@@ -259,6 +259,46 @@ tests:
...
+- as: gcp-ipi-compact-filestore-csi-f14
+  cron: 43 19 4,24 * *
+  steps:
+    cluster_profile: gcp-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    dependency_overrides:
+      OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.15
+    env:
+      COMPUTE_NODE_TYPE: t2a-standard-4
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-stable.yaml:271]: For compact/sno cluster, need a bigger size of master nodes, but does not need compute nodes.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml [hunk 2]
```diff
@@ -761,6 +949,34 @@ tests:
...
+    - chain: openshift-e2e-test-qe-destructive
+    workflow: cucushift-installer-rehearse-gcp-ipi-sno
+- as: gcp-ipi-sno-arm-f14
+  cron: 32 13 6,16 * *
+  steps:
+    cluster_profile: gcp-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      COMPUTE_NODE_TYPE: t2a-standard-4
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml:972]: For sno cluster, need a bigger size of master nodes, but does not need compute nodes, refer to `single-node-conf-gcp` step.

Comment (lwan-wanglin) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml:972]: ok, also see some tag issue(set to 4.15 for 4.14 jobs), will fix togather

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-stable.yaml (hunk 1 above)
Comment (lwan-wanglin) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-stable.yaml]: removed compute. 
now ipi-conf-gcp can't cover compact for arm after some pr merge, see [here](https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/gcp/ipi-conf-gcp-commands.sh#L24-L39) I will fix it in another pr.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml (hunk 2 above)
Comment (lwan-wanglin) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml]: fixed

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-stable.yaml (hunk 1 above)
Comment (lwan-wanglin) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-stable.yaml]: fix pr is here: https://github.com/openshift/release/pull/45543

**Comment by jianlinliu** [comment]
cc @liangxia to review.

**Pull request merged**